src/app/forecasting/input/ynab/ynab.component.ts

```diff
@@ -179,6 +179,54 @@ export class YnabComponent implements OnInit {
     }
     return result;
   }
+  
+  /* Chooses months based on common decisions. Options: All, Last calendar year, Last 12 months, Year to date, Current month. */
+  async quickChooseMonths(choice: string) { 
+    // Get some etc facts that are shared by some of the buttons below.
+    const budgetId = window.localStorage.getItem('br4-selected-budget');
+    const currentMonth = await this.ynabService.getMonth(budgetId, 'current');

```

Comment:
Future improvement would be to cache this. Pretty sure we already make this call.